controllers/user_controller.go

```diff
@@ -0,0 +1,65 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/drift-org/backend/models"
+	"github.com/gin-gonic/gin"
+	"github.com/kamva/mgm/v3"
+)
+
+type UserController interface {
+	Info(context *gin.Context)
+}
+
+type userController struct {
+}
+
+func NewUserController() UserController {
+	return &userController{}
+}
+
+func (ctrl *userController) Info(context *gin.Context) {
+	// Id refers to MongoDB user object ID. Specificty refers to the verbocity of the returned friends array
+	// 0 returns ONLY friend user IDs, 1 returns friend user IDs AND usernames
+	type ICreate struct {
```

Comment:
copy and paste error. `ICreate` is for creating  (CRUD) routes

Reply:
Actually, now that I think about it, we don't usually have a body for GET routes (at least for presoroban). These would be query parameters if we followed that. Let's discuss at meeting.